src/components/cp-art/zombie/Zombie.tsx

```diff
@@ -0,0 +1,34 @@
+import ZombieImg from "../../../assets/cp-images/zombie.png"
+import Circle from "../../../assets/cp-doodles/circle.png"
+import X from "../../../assets/cp-doodles/x.png"
+import Spiral from "../../../assets/cp-doodles/spiral.png"
+import Baloon from "../../../assets/cp-doodles/baloon.png"
+import Wave from "../../../assets/cp-doodles/wave.png"
+import Hash from "../../../assets/cp-doodles/hash.png"
+import './Zombie.css'
+
+interface ZombieProps {
+  className?: string
+}
+
+export function Zombie({ className }: ZombieProps) {
+  const classes = `${className ? className : 'zombie'}`
+  
+  return (
+    <div className={classes}>
+      <img className="zombie-img" src={ZombieImg} alt="Image" />
```

Comment:
Acho que seria legal incluir as outras imagens para parametrizar por cor nas props (se quiser posso fazer depois pq é bem simples)

Reply:
Okayyy, conforme a gnt for usando outras cores eu vou add